pkg/pow/miningTasker.go

```diff
@@ -60,7 +60,8 @@ func CreateTasker(logger log.Logger, cfg *config.Config, proxy db.DataServerProx
 }
 
 func (mt *MiningTasker) GetWork() (*Work, bool) {
-	dispKey := mt.pubKey + "-" + db.DisputeStatusKey
+	// TODO: dispKey for multi key setup.
+	dispKey := mt.pubKey + "-" + db.DisputeStatusPrefix
```

Comment:
What do you mean by this comment? Each mining manager uses a single pub/priv key so what do you mean by `multi key setup` ?

Also to make it consistent everwhere can we have the prefix first
```
 db.DisputeStatusPrefix + "-" +  mt.pubKey
 ```

Reply:
Oh, actually got it now!